docs/sources/reference/components/prometheus/prometheus.remote_write.md

```diff
@@ -404,6 +404,37 @@ prometheus.remote_write "default" {
 }
 ```
 
+## Troubleshooting
+
+### Out of order errors
+
+You may sometimes see an "out of order" error in the {{< param "PRODUCT_NAME" >}} log files.
+This means that {{< param "PRODUCT_NAME" >}} sent a metric sample which has an older timestamp than a sample which the database already ingested.
```

Comment:
```suggestion
This means that {{< param "PRODUCT_NAME" >}} sent a metric sample that has an older timestamp than a sample that the database already ingested.
```

The easy way to decide if you should use which or that is if you can remove the part of the sentence that you are writing, it's probably "which", but if you can't remove the part of the sentence, then it's probably "that".

My kitchen, which is in the front of the house, is painted yellow.
vs.
The presentation that you gave yesterday was excellent.